src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/cell/boolean-label-cell.js

```diff
@@ -18,8 +18,8 @@ define(['oro/datagrid/string-cell', 'oro/translator'],
                     return this;
                 }
 
-                var status = 'true' === value ? 'success' : 'important';
-                var label = 'true' === value ? 'Yes' : 'No';
+                var status = (true === value || '1' === value) ? 'success' : 'important';
```

Comment:
But maybe there is another case, and we need the three of them: true, 'true' and '1'
#viandedatagrid